io.github.sigmasd.recordapps.yml

```diff
@@ -0,0 +1,75 @@
+  - --share=network # local server
+modules:
+  - name: recordapps
+    buildsystem: simple
+    build-commands:
+      - DENORT_BIN=$PWD/denort ./deno compile -o record -A --include dist/ dist/webview.ts
+      - install -Dm755 record /app/bin/record
+      - install -Dm755 run.sh /app/bin/run.sh
+      - install -D libwebview.*.so -t /app/bin/
+
+      - install -D distro/io.github.sigmasd.recordapps.desktop -t /app/share/applications/
+      - install -D distro/io.github.sigmasd.recordapps.metainfo.xml -t /app/share/metainfo/
+
+      - install -D distro/icons/64x64/io.github.sigmasd.recordapps.png -t /app/share/icons/hicolor/64x64/apps/
+      - install -D distro/icons/128x128/io.github.sigmasd.recordapps.png -t /app/share/icons/hicolor/128x128/apps/
+      - install -D distro/icons/256x256/io.github.sigmasd.recordapps.png -t /app/share/icons/hicolor/256x256/apps/
+
+    sources:
+      # app
+      - type: archive
+        url: https://github.com/sigmaSd/recordapps/archive/refs/tags/1.5.2.tar.gz
+        sha256: c7ea4ae17a936d77b39c6e613af485626453450189a1a3df00c39fdceff16db9
+
+      # deno
+      - type: archive
+        url: https://github.com/denoland/deno/releases/download/v2.1.7/deno-x86_64-unknown-linux-gnu.zip
+        sha256: 8ea47ace85a2daadb60165d2aa519e11aef023d74441a24c4dc1d106dffdd19b
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/denoland/deno/releases/download/v2.1.7/denort-x86_64-unknown-linux-gnu.zip
+        sha256: f00adc493667d5e414165d102d537023109dbd35d62b254566caa1485e0d2bd0
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://github.com/denoland/deno/releases/download/v2.1.7/deno-aarch64-unknown-linux-gnu.zip
+        sha256: fb37f230b55898660807247ea28a37afead40bc3713e2ee022a5c1e9476ac9aa
+        only-arches:
+          - aarch64
+      - type: archive
+        url: https://github.com/denoland/deno/releases/download/v2.1.7/denort-aarch64-unknown-linux-gnu.zip
+        sha256: 0773c54ab7a76658a6d32067cec41d87f2845f1eaca1a057a35ad2590028e8e9
+        only-arches:
+          - aarch64
+
+      # libwebview ffi library
+      - type: file
+        url: https://github.com/sigmaSd/webview_deno/releases/download/0.9.2/libwebview.x86_64.so
+        sha256: 5a66b75e14e9360d5c24ba53574e5ed5b1b476843623721aae8fc47a15a69a42
+        only-arches:
+          - x86_64
+      - type: file
+        url: https://github.com/sigmaSd/webview_deno/releases/download/0.9.2/libwebview.aarch64.so
+        sha256: e9ee12c1fe6f0c587c37cca827586bf5c202fd7e04afcd01b780c993878d132a
+        only-arches:
+          - aarch64
+
+      # run script
+      - type: script
+        dest-filename: run.sh
+        commands:
+          # https://discourse.gnome.org/t/how-to-fix-evolution-not-starting-after-update/25389
+          - WEBKIT_DISABLE_SANDBOX_THIS_IS_DANGEROUS=1 PLUGIN_URL=${FLATPAK_DEST}/app/bin exec record "$@"
```

Comment:
`$FLATPAK_DEST` doesn't exist at run time.

Reply:
removed it